explorer/client/src/utils/authenticationStorage.ts

```diff
@@ -0,0 +1,21 @@
+import storage from 'local-storage-fallback'
+
+const ADMIN_ALLOWED = 'explorer.adminAllowed'
+
+export function getAdminAllowed(): boolean {
+  const adminAllowed: string | null = storage.getItem(ADMIN_ALLOWED)
+
+  if (adminAllowed) {
+    try {
+      return JSON.parse(adminAllowed)
+    } catch {
+      // Fall through
```

Comment:
This could be hard to debug if we run into an issue here, should we `console.error` at the very least?

Reply:
Good call